examples/etcd/resources/monitor-job.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: appcontroller.k8s/v1alpha1
+kind: Definition
+metadata:
+  name: monitor-clustername-job
+meta:
+  timeout: 0
+job:
+  apiVersion: batch/v1
+  kind: Job
```

Comment:
In the other hand, it's a job which is supposed to keep your ETCD cluster alive. So technically it's a job.

I'd be more concern about: 

> Note that even if you specify .spec.parallelism = 1 and .spec.completions = 1 and .spec.template.spec.restartPolicy = "Never", the same program may sometimes be started twice.

from docs.